internal/config/config.go

```diff
@@ -33,7 +33,9 @@ type MQTTConfig struct {
 }
 
 type StandbyConfig struct {
-	BackupFile string `yaml:"backup_file" default:"plan.json"`
+	BackupFile     string `yaml:"backup_file" default:"plan.json"`
+	CheckInterval  int    `yaml:"check_interval" default:"60"`
+	OutageInterval int    `yaml:"outage_interval" default:"120"`
```

Comment:
These names are confusing, any suggestions for improvement? `CheckInterval` is the number of seconds to wait in between checks for commands (i.e. the periodicity of the ticker), and `OutageInterval` is the number of seconds to wait since the last command was received, before switching into 'command mode'

Reply:
1. Other options: `OutageCheckPeriodicity`, `OutageDurationThreshold`.  
2. `aconfig` is capable of reading `time.Duration` natively: https://github.com/cristalhq/aconfig/blob/main/aconfig_test.go#L166. That would be a bit clearer, as it we won't need to guess the units.

Reply:
`Threshold` is what I was looking for, thanks! And using duration as the config type is an excellent idea